### ISSUE TITLE
Support for multiple environments in template yaml

### ISSUE THREAD

**Issue by khushbulohia** [oneops/boo#45]
The feature should be backward compatible. Existing single environment yaml template should continue to work 
To add more then one environment config in template yaml, follow below format. Make sure old environment config is removed from the yaml file

```
environments: # Only support one environment for now.
  dev:
    global_dns: 'true'
    availability: 'redundant'
    platforms:
      tomcat:
        auto_healing:
          autorepair: false
          autoreplace: true
          replace_after_minutes: 60
          replace_after_repairs: 4
        scale: # Don't change this
          compute: # Component name
            current: '3' # Currently set current=min=max as the total number of computes
            min: '1'
            max: '5'
        components:
          lb:
            ecv_map: '{"8080":"GET /"}'
    clouds:
      {{cloud}}:
        priority: '1'
        dpmt_order: '1'
        pct_scale: '100'
  dev0:
    global_dns: 'true'
    availability: 'single'
    platforms:
      tomcat:
        auto_healing:
          autorepair: true
          autoreplace: true
          replace_after_minutes: 40
          replace_after_repairs: 2
    clouds:
      stub-dfw2a:
        priority: '1'
        dpmt_order: '1'
        pct_scale: '100'
      stub-dfw2b:
        priority: '2'
        dpmt_order: '1'
        pct_scale: '100'
  dev1:
    global_dns: 'true'
    availability: 'redundant'
    platforms:
      tomcat:
        auto_healing:
          autorepair: true
          autoreplace: false
          replace_after_minutes: 10
          replace_after_repairs: 1
        components:
          lb:
            ecv_map: '{"8080":"GET /paas-perf"}'
    clouds:
      {{cloud}}:
        priority: '1'
        dpmt_order: '5'
        pct_scale: '100'```

**Comment by khushbulohia** [comment]
Sample yaml file added test-tomcat.yaml in test resources with multiple environments

**Comment by khushbulohia** [comment]
https://github.com/oneops/boo/pull/44 PR created

**Comment by khushbulohia** [comment]
Released 1.0.14
